src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -154,6 +159,22 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 				this._undoStack.maxStackSize = newStackSize;
 				this._redoStack.maxStackSize = newStackSize;
 			}
+			if (e.affectsConfiguration('notebook.renderTablesInHtml')) {
+				this._htmlMarkdownConverter = this._instantiationService.createInstance(HTMLMarkdownConverter, this.notebookUri);
+				this.updateCellSource();
+				if (this.isEditMode) {
```

Comment:
Are you not able to refresh the edit view as well?

Reply:
what do you mean with edit view? We only need to call updateModel if the cell is already in edit mode. We call the updateModel when editing a cell.

Reply:
Why do we only do this if a cell is in edit mode? We should be refreshing all cells regardless of whether they're being edited or not